libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -352,7 +352,15 @@ - (void)testPlaintextToEncryptedAccountUpgrade {
     XCTAssertTrue([fm fileExistsAtPath:userFilePath], @"User directory for User ID '%@' and Org ID '%@' should exist.", userId, orgId);
     
     // Verify that the user data is now encrypted on the filesystem.
-    XCTAssertThrows([NSKeyedUnarchiver unarchiveObjectWithFile:userFilePath], @"User account data for User ID '%@' and Org ID '%@' should now be encrypted.", userId, orgId);
+    @try {
```

Comment:
Since unarchive returns nil instead of throwing an exception on iOS9, we need to tweak the test so that it passes both on iOS9 and on older versions